ports/esp32/Makefile

```diff
@@ -151,6 +152,7 @@ SRC_C = \
 	modesp32.c \
 	moduhashlib.c \
 	espneopixel.c \
+	machine_ulp.c \
```

Comment:
Putting the ULP class in the esp32 module would mean this file becomes `esp32_ulp.c`.

Reply:
I agree with you, the esp module is probably a better place for this code, I will move it.